src/coreclr/jit/lclvars.cpp

```diff
@@ -1710,6 +1710,12 @@ void Compiler::lvaClassifyParameterABI(Classifier& classifier)
         {
             wellKnownArg = WellKnownArg::RetBuffer;
         }
+#ifdef TARGET_WASM
+        else if (i == info.compThisArg)
+        {
+            wellKnownArg = WellKnownArg::ThisPointer;
+        }
+#endif //TARGET_WASM
```

Comment:
```suggestion
```
No longer needed now that we've decided on silencing out the assert.

Reply:
Right, wasn't sure if it was "correct".  Now removed.